Translation corrections (#112)
07th-mod/higurashi-console-arcs: Update/some_009.txt

			NULL, "\"That's right.", Line_WaitForInput);
	ModPlayVoiceLS(4, 28, "ps3/s04/28/310500279", 256, TRUE);
	OutputLine(NULL, "ただ、容疑者が精神的に錯乱していたのは目撃証言から考えても事実のようだし、",

> 			NULL, " However, the suspect's mental derangement appears to be fact when considering the eyewitness testimonies, ", GetGlobalFlag(GLinemodeSp));

Reviewer: ```suggestion
			NULL, " However, the suspect's mental derangement seems to be corroborated by eyewitness testimony, ", GetGlobalFlag(GLinemodeSp));
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#9d7f69>巴</color>", NULL, "<color=#9d7f69>Tomoe</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 28, "ps3/s04/28/310500286", 256, TRUE);
	OutputLine(NULL, "…つい先日の大災害のときも、オヤシロさまの祟りだってお年寄りたちが騒いでいたみたいだけど、あれってどういうこと？」",

> 			NULL, "...The elderly have apparently been blaming Oyashiro-sama's curse for the disaster occurred the other day. What does that mean?\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "...The elderly have apparently been blaming Oyashiro-sama's curse for the disaster that occurred the other day. What does that mean?\"", Line_Normal);
```
